openerp/addons/base/ir/ir_ui_menu.py

```diff
@@ -337,8 +338,8 @@ def load_menus_root(self, cr, uid, context=None):
 
 
     @api.cr_uid_context
-    @tools.ormcache_context('uid', keys=('lang',))
-    def load_menus(self, cr, uid, context=None):
+    @tools.ormcache_context('uid', 'debug', keys=('lang',))
+    def load_menus(self, cr, uid, debug=False, context=None):
```

Comment:
Why adding an unused parameter ? Moreover new parameters in critical methods like this should always be added at the end, for backward compatibility.